packages/edit-site/src/components/post-edit/index.js

```diff
@@ -43,11 +56,20 @@ function PostEditForm( { postType, postId } ) {
 								ids[ 0 ]
 						  )
 						: null,
+				records:
+					ids.length > 1
+						? ids.map( ( id ) =>
+								select( coreDataStore ).getEditedEntityRecord(
+									'postType',
+									postType,
+									id
+								)
+						  )
+						: null,
```

Comment:
I do seem to get this warning when adding the above, which I am not entirely sure why. 
`The 'useSelect' hook returns different values when called with the same state and parameters. This can lead to unnecessary rerenders.`

Reply:
Or is there a better way to get the edited record for multiple entities?